scripts/lint-style.lean

```diff
@@ -129,6 +145,7 @@ def lintStyleCli (args : Cli.Parsed) : IO UInt32 := do
   let mut numberErrors ← lintModules nolints allModuleNames style fix
   if ← checkInitImports then numberErrors := numberErrors + 1
   if !(← allScriptsDocumented) then numberErrors := numberErrors + 1
+  if !(← checkModulesUpperCamelCase allModuleNames) then numberErrors := numberErrors + 1
```

Comment:
This is an idea for a wider refactor: how about each `check` returns an `IO Nat`, rather than an `IO Bool` and this command simply adds them all up?  When there are no reports, the number is `0`, otherwise it may be `1` (or whatever is reasonable).

Then, you do not have `if`s, you simply accumulate all values and continue as you already do.

Reply:
Sure, why not? Let me implement that (tomorrow)!

Reply:
I just tried this, and it's much clearer: all three checks can naturally report their number of errors. (And this design makes it impossible to inadvertedly flip the meaning of a boolean, so is a clear improvement!)